templates/campaigns.html

```diff
@@ -83,6 +83,10 @@ <h4 class="modal-title" id="campaignModalLabel">New Campaign</h4>
               <input type="text" class="form-control" placeholder="http://192.168.1.1" id="url"/>
 	      <label class="control-label" for="url">Schedule: </label>
               <input type="text" class="form-control" id="launch_date"/>
+          <label class="control-label" for="senddelay">Min delay time: <i class="fa fa-question-circle" data-toggle="tooltip" data-placement="right" title="Minimum randomized delay time between sending of each campaign email"></i></label>
```

Comment:
Think, I'm going to figure this out another day... it's getting late now and I have to be somewhere tomorrow.  Maybe early next week or something.